<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>makeTran</class>
 <widget class="QDialog" name="makeTran">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;
font-family: inter;</string>
   </property>
   <property name="text">
    <string>Transaction:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_transaction">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>15</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;
font-size: 16px;
font-family:inter;
color:black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Where are you transacting?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>65</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;
font-family: inter;</string>
   </property>
   <property name="text">
    <string>Amount:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_amount">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>60</y>
     <width>281</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;
font-size: 16px;
font-family:inter;
color:black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>How much would you like to transact?</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_category">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>104</y>
     <width>281</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;
font-size: 16px;
font-family:inter;
color:black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>What category is the transaction?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>107</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;
font-family: inter;</string>
   </property>
   <property name="text">
    <string>Category:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>15</y>
     <width>82</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
